float64 t0
float64 tf
float64 tstep

float64[] posx_traj
float64[] velx_traj
float64[] accx_traj

float64[] posy_traj
float64[] vely_traj
float64[] accy_traj

float64[] posz_traj
float64[] velz_traj
float64[] accz_traj

